src/aap_eda/core/validators.py

```diff
@@ -68,3 +69,16 @@ def check_awx_tokens(user_id: int) -> int:
         )
 
     return user_id
+
+
+def is_extra_var_dict(extra_var: str):
+    try:
+        data = yaml.safe_load(extra_var)
+        if not isinstance(data, dict):
+            raise serializers.ValidationError(
+                "Extra var must be in JSON or YAML format."
```

Comment:
@Dostonbek1 I think this should the data should be in object format, with key=value format.
Also we might have to catch Parsing errors, that would be a different error. e.g a data like 2 which is an integer